docs/source/installation/login.md

```diff
@@ -46,43 +54,45 @@ All QHub users will be part of the `qhub` realm (a realm is a distinct identity
 
 The `qhub` realm is selected by default, we strongly recommend leaving it as is.
 
+Steps to create a new user:
+
 1. Click 'Users' along the left-hand side of the page.
 
 2. Click the 'Add user' button and you will see the new user form:
 
-![Keycloak add user tab screenshot - new user form ](../images/keycloak_add_users.png)
+    ![Keycloak add user tab screenshot - new user form](../images/keycloak_add_users.png)
 
-3. Fill out the three fields, outlined above. These are 'Username', 'Email', and 'Groups'.
+3. Fill out the three fields outlined above. These are 'Username', 'Email', and 'Groups'. (We explain these fields below.) Then click save.
 
-Depending on the authentication provider selected ('password', 'GitHub' or 'Auth0'), the values entered into the 'Username' field will differ slightly. The following table outlines
+4. Username. Depending on the authentication provider selected ('password', 'GitHub' or 'Auth0'), the values entered into the 'Username' field will differ slightly. The following table outlines
 those differences:
 
-|          | Password          | GitHub            | Auth0                 |
-| -------- | ----------------- | ----------------- | --------------------- |
-| Username | *unique username* | *GitHub username* | *Email to login with* |
+    |          | Password          | GitHub            | Auth0                 |
+    | -------- | ----------------- | ----------------- | --------------------- |
+    | Username | *unique username* | *GitHub username* | *Email to login with* |
 
-Once the 'Username' field is updated, please add a valid email address in the 'Email' field.
+5. Once the 'Username' field is updated, please add a valid email address in the 'Email' field.
 
-> NOTE: Although not required, users may not be able to log into Grafana if this field isn't properly set.
+    > NOTE: Although not required, users may not be able to log in to Grafana if this field isn't properly set.
 
-Lastly, associate the user with one or more of the 'Groups'. Out of the box, QHub is deployed with the following groups: 'admin', 'analyst', and 'developer' (see the
+6. Lastly, associate the user with one or more of the 'Groups'. Out of the box, QHub is deployed with the following groups: 'admin', 'analyst', and 'developer' (see the
 [Groups](./login.md#groups) section below for more details).
 
-Enter the name you would like for the user then click 'Save'.
+7. Click save.
```

Comment:
To be honest, I don't love the indentation that these markdown lists create. I wonder if it would be better to reformat as:

```md
Step 1. Blah blah blah

![some screenshot](url-to-screenshot.jpeg)

Step 2.
```

in order to avoid the indentation.

Reply:
feel free to adopt it, we can always switch back if needed :slightly_smiling_face: